books/schemas/types.py

```diff
@@ -1,19 +1,47 @@
+from typing import Optional
+
 import graphene # type: ignore
+from graphene.types.datetime import Date # type: ignore
+
+from books.entities import Author, Book
+
+
+class BookAuthorType(graphene.ObjectType):
+    id = graphene.ID()
 
+    first_name = graphene.String()
+    last_name = graphene.String()
+    full_name = graphene.String()
```

Comment:
full_name of what? Book?

Reply:
Full_name of author :D

Reply:
And yes it is related to the resolve_full_name because it's maps the function to the name in schema